actors/builtin/miner/miner_actor.go

```diff
@@ -1363,24 +1363,33 @@ func (a Actor) ReportConsensusFault(rt Runtime, params *ReportConsensusFaultPara
 	}
 
 	// Reward reporter with a share of the miner's current balance.
-	slasherReward := RewardForConsensusSlashReport(faultAge, rt.CurrentBalance())
-	_, code := rt.Send(reporter, builtin.MethodSend, nil, slasherReward)
-	builtin.RequireSuccess(rt, code, "failed to reward reporter")
-
 	var st State
 	rt.State().Readonly(&st)
+	unlockedBalance := st.GetUnlockedBalance(rt.CurrentBalance())
+	slasherReward := RewardForConsensusSlashReport(faultAge, rt.CurrentBalance())
+	if slasherReward.GreaterThan(unlockedBalance) {
+		rt.Abortf(exitcode.ErrInsufficientFunds, "insufficient unlocked balance to pay reward")
```

Comment:
Blocking: I don't think we want to prevent the reporting of a consensus fault, even if no reward can be paid.